locale/id/LC_MESSAGES/django.po

```diff
@@ -258,6 +279,10 @@ msgstr "Konfirmasi Sandi"
 msgid "Change"
 msgstr "Ubah "
 
+#: gem/templates/profiles/viewprofile.html:72
+msgid "Your gender is: "
```

Comment:
I really don't like the whitespace in trans strings, is this just me? It feels like an antipattern?